lua/entities/gmod_wire_coldetector.lua

```diff
@@ -1,27 +1,16 @@
 AddCSLuaFile()
 
-if !WireLib then return end
 DEFINE_BASECLASS( "base_wire_entity" )
 ENT.PrintName       = "Wire Collision Detector"
 ENT.WireDebugName	= "Collision Detector"
-local f =""
-local i = 1
-/*if CLIENT then
-	while(true) do
-		name, value = debug.getupvalue(effects.Create,i)
-		if name == nil then break end
-		if name == "EffectList" then
-			PrintTable(value)
-		end
-		i=i+1
-	end
-end*/
 
 if CLIENT then return end
+
 local stickplayer=CreateConVar( "wire_col_detector_stickplayer", 0, 0, "Allow linked entity stick to players" )
 local stickprops=CreateConVar( "wire_col_detector_stickprops", 0, 0, "Allow sticking to other players stuff" )
+
 function entitiesCollide(ent,data)
-	if !IsValid(ent.ColSensor) || ent.ColSensor.LinkedEnt != ent then return end
+	if !IsValid(ent.ColSensor) or ent.ColSensor.LinkedEnt != ent then return end
```

Comment:
Do you guys think that it's better: 
"!IsValid"  to "not IsValid"
"!="          to "~="

To avoid using Garry's pre-processor. I always avoid it while coding in GLua